packages/@aws-cdk/aws-route53-patterns/README.md

```diff
@@ -0,0 +1,34 @@
+# Route53 Patterns for the CDK Route53 Library
+<!--BEGIN STABILITY BANNER-->
+
+---
+
+![Stability: Experimental](https://img.shields.io/badge/stability-Experimental-important.svg?style=for-the-badge)
+
+> **This is a _developer preview_ (public beta) module. Releases might lack important features and might have
+> future breaking changes.**
+>
+> This API is still under active development and subject to non-backward
+> compatible changes or removal in any future version. Use of the API is not recommended in production
+> environments. Experimental APIs are not subject to the Semantic Versioning model.
+
+---
+<!--END STABILITY BANNER-->
+
+This library contains commonly used patterns for Route53.
+
+
+## HTTPS Redirect
+
+This construct allows creating a simple HTTP->HTTPS and domainA->domainB redirect using CloudFront and S3.
+
```

Comment:
Add an snippet on how to create an HTTP=>HTTPS redirect. It's not clear from this.

Reply:
I will reword this sentence. It is doing a domain redirect and http->https redirect for domainA. You cannot use this construct to redirect http->https on your app domain, as you can only have of DNS entry for your domain.